RetsTalk/RetsTalk/Retrospect/Controller/RetrospectListViewController.swift

```diff
@@ -157,14 +168,22 @@ final class RetrospectListViewController: BaseViewController {
         Task {
             await retrospectManager.fetchRetrospects(of: [.pinned, .inProgress, .finished])
             sortAndSendRetrospects()
+            isRetrospectAppendable = true
         }
     }
     
     private func fetchPreviousRetrospects() {
         Task {
-            await retrospectManager.fetchPreviousRetrospects()
+            let appendedCount = await retrospectManager.fetchPreviousRetrospects()
+            guard appendedCount != 0
+            else {
+                isRetrospectAppendable = false
+                isRetrospectFetching = false
+                return
+            }
+            
```

Comment:
```suggestion
            let appendedCount = await retrospectManager.fetchPreviousRetrospects()
            isRetrospectFetching = false
            guard appendedCount != 0
            else {
                isRetrospectAppendable = false
                return
            }
            
```
으로도 사용 가능할 것 같아요! 밑에 false도 뺄 수 있고